studybits-student/src/main/java/nl/quintor/studybits/student/model/Claim.java

```diff
@@ -0,0 +1,18 @@
+package nl.quintor.studybits.student.model;
```

Comment:
Claim is used in the wrapper as a "resolved" claim, i.e. one that has been fulfilled. Is that the future purpose of this class? (If not, maybe rename to ClaimOfferRecord)